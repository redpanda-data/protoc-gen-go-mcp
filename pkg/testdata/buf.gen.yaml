version: v2
managed:
  enabled: true
  override:
    - file_option: go_package_prefix
      value: github.com/redpanda-data/protoc-gen-go-mcp/pkg/testdata/gen/go
  disable:
    - file_option: go_package
      module: buf.build/googleapis/googleapis
plugins:
  - local: protoc-gen-go
    out: gen/go
    opt: paths=source_relative
  - local: protoc-gen-go-grpc
    out: gen/go
    opt: paths=source_relative
  - local: protoc-gen-connect-go
    out: gen/go
    opt: paths=source_relative
  - local: ["go", "run", "../../cmd/protoc-gen-go-mcp"]
    out: gen/go
    opt: paths=source_relative