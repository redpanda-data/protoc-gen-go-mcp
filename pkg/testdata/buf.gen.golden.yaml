version: v2
managed:
  enabled: true
  override:
    - file_option: go_package_prefix
      value: github.com/redpanda-data/protoc-gen-go-mcp/pkg/testdata/golden
  disable:
    - file_option: go_package
      module: buf.build/googleapis/googleapis
plugins:
  - local: protoc-gen-go
    out: golden
    opt: paths=source_relative
  - local: ["go", "run", "../../cmd/protoc-gen-go-mcp"]
    out: golden
    opt: paths=source_relative